New Timetracking documentation (#160)
Enalean/tuleap-documentation-en: languages/en/user-guide/timetracking.rst

   :name: Timetracking artifact times view

.. attention::
   The format to add a time is hh:mm

User widgets
------------

> Timetracking plugin also provides user widgets :

Reviewer: It's unclear to what this refer to. I suggest you put a list with the list of widgets with anchor links to the dedicated sections.

Personal Time Tracking
~~~~~~~~~~~~~~~~~~~~~~

Timetracking plugin also provides a user widget. This widget allows users to filter and display times added in the whole
platform during the time period provided. By default, the search is done for the last week.

If multiple times are added in the same artifact in this time period, the widget will only show one entry per artifact
summing the total time for this period.

.. figure:: ../images/screenshots/timetracking/timetracking-personal-widget.png
   :align: center
   :alt: Timetracking user widget
   :name: Timetracking user widget

Detailed times modal
''''''''''''''''''''

> Artifact's informations are displayed on the top of the modal. Artifact's times of the given period are displayed on the table.

Reviewer: "information" is an uncountable noun in English.

It's possible to add and edit times on this modal.

.. figure:: ../images/screenshots/timetracking/timetracking-personal-user-widget-modal.png
   :align: center
   :alt: Timetracking personal user widget modal
   :name: Timetracking personal user widget modal

Timetracking Overview
~~~~~~~~~~~~~~~~~~~~~~

This widget allows users to filter and display times added in the selected trackers in the time period provided. By default, the search is done for the last month.

> Times are grouped by trackers. You must have the reading right to select a tracker. The sum of selected trackers' times is displayed.

Reviewer: ```suggestion
Times are grouped by trackers. You must be able to access a tracker to select it.
```
